commit 16864159654cb6c77b3db36dd218e4103e3bd0c2
Author: Gwenole Beauchesne <gbeauchesne@splitted-desktop.com>
Date:   Thu Jun 3 09:23:45 2010 +0200

    Don't install test programs.

Rebased-for-1.0.14-by: Thomas Backlund <tmb@mageia.org>

 test/decode/Makefile.am     |    2 +-
 test/encode/Makefile.am     |    2 +-
 test/putsurface/Makefile.am |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff -Nurp libva-1.0.14.orig/test/decode/Makefile.am libva-1.0.14/test/decode/Makefile.am
--- libva-1.0.14.orig/test/decode/Makefile.am	2011-07-28 03:38:03.000000000 +0300
+++ libva-1.0.14/test/decode/Makefile.am	2011-08-30 18:30:38.594046581 +0300
@@ -20,7 +20,7 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-bin_PROGRAMS = mpeg2vldemo loadjpeg
+check_PROGRAMS = mpeg2vldemo loadjpeg
 
 INCLUDES = \
 	-I$(top_srcdir)				\
diff -Nurp libva-1.0.14.orig/test/encode/Makefile.am libva-1.0.14/test/encode/Makefile.am
--- libva-1.0.14.orig/test/encode/Makefile.am	2011-07-28 03:38:03.000000000 +0300
+++ libva-1.0.14/test/encode/Makefile.am	2011-08-30 18:33:29.046207110 +0300
@@ -20,9 +20,9 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-bin_PROGRAMS = avcenc
+check_PROGRAMS = avcenc
 if USE_X11
-bin_PROGRAMS += h264encode
+check_PROGRAMS += h264encode
 endif
 
 INCLUDES = \
diff -Nurp libva-1.0.14.orig/test/putsurface/Makefile.am libva-1.0.14/test/putsurface/Makefile.am
--- libva-1.0.14.orig/test/putsurface/Makefile.am	2011-07-28 03:38:03.000000000 +0300
+++ libva-1.0.14/test/putsurface/Makefile.am	2011-08-30 18:30:38.594046564 +0300
@@ -20,7 +20,7 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-bin_PROGRAMS = putsurface
+check_PROGRAMS = putsurface
 
 INCLUDES = \
 	-I$(top_srcdir)		\
